src/jvmMain/kotlin/io/beatmaps/login/main.kt

```diff
@@ -76,8 +76,10 @@ data class Session(
     val oauth2ClientId: String? = null,
     val suspended: Boolean = false
 ) {
+    constructor(userId: Int, hash: String?, userEmail: String, userName: String, testplay: Boolean, steamId: Long?, oculusId: Long?, admin: Boolean, uniqueName: String?, canLink: Boolean, alerts: Int?, curator: Boolean, suspended: Boolean) :
```

Comment:
Nope, you need to upgrade old sessions. They won't have your new field. They will have all the old fields.